cgt_calc/model.py

```diff
@@ -225,7 +225,7 @@ def __init__(
         bed_and_breakfast_date_index: datetime.date | None = None,
         spin_off: SpinOff | None = None,
         dividend: Dividend | None = None,
-        eri: ExcessReportedIncome | None = None,
+        eris: tuple[ExcessReportedIncome, ...] = (),
```

Comment:
Why not `list`?

Reply:
I think list is semantically more correct for collections which store elements of the same type.

Reply:
Fair enough be ready for all the `or []`